### DISCUSSION TITLE
Issue with automatic mesh generation for selected segmentations in Neuroglancer

### DISCUSSION THREAD

**Discussion by yuxuandexter** [google/neuroglancer#626] [Q&A] [answered]
![Screenshot from 2024-07-26 11-28-35](https://github.com/user-attachments/assets/741decf3-4c6f-488a-b966-233046d994ca)
I load h5 file from the local directory to the neuroglancer, the neuroglancer could load all instance segmentations but it failed to automatically generate a mesh.

**Comment by yuxuandexter** [comment]
After further debugging, I found out that the neuroglancer can not support 32 bit automatic mesh generation in the default. Is there any way to solve this problem?

**Comment by jbms** [comment]
You need to change the layer type to `segmentation` --- click on where it says `img` next to the layer name and select segmentation.

**Reply by yuxuandexter** [reply]
Thanks for your reply. I figured out my minor mistake. However, I still want to know whether neuroglancer support 32 bit automatic mesh generation. 
![image](https://github.com/user-attachments/assets/8a8a2ca2-2d33-4403-803d-ed6c6afaae82)
Based on this image, I double click the selected segmentation, which fails to generate a 3D mesh. My dataset is 32 bit for segmentation. Is there any available way to solve this problem?

**Reply by jbms** [reply, marked as answer]
int32 isn't supported but uint32 is.

**Reply by yuxuandexter** [reply]
Thank you so much. One last question. Do you know where is the neuroglancer python API documentation? If I want to add proofreading function, what type of resources will be helpful for me to do it!
